lib/helpers.coffee

```diff
@@ -33,3 +33,53 @@ module.exports =
         return syntax[0]
 
     return existingSyntax
+
+  ###*
+   * Attempts to resolve the root directory/project directory of the currently open file/editor instance
+   * @param {string} filePath - The currently active editor file path
+   * @return {Object|null} The project directory instance or null if no project root is found
+   ###
+  getRootDir: (filePath) ->
+    rootDir = null
+    atom.project.rootDirectories.forEach((dir) ->
+      if dir.contains(filePath) then rootDir = dir
+    )
+    return rootDir
+
+  ###*
+   * Checks to see if a config file exists in the projects root directory if a root directory exists
+   * @param {string|null} dir - The current project root or null if a project doesn't exist
+   * @param {string} configExt - The Sass-lint config extension
+   * @return {string|null} The path to the config file if located in the project root, null if it doesn't exist
+   ###
+  getRootDirConfig: (dir, configExt) ->
+    fs = require 'fs'
+    rootDir = dir
+
+    if rootDir
+      rootDir = rootDir.getPath() + '/' + configExt
```

Comment:
Shouldn't this just be `rootDir`? Also use `path.join()` instead of concatenating in a `/` manually.